### PR TITLE
BaseStyles: make it possible to change the default font

### PR DESCRIPTION
**Pull request by bpierre**
By setting the `fontFamily` prop.

Related to https://github.com/aragon/aragon/issues/490

### CODE REVIEW

File: src/components/BaseStyles/BaseStyles.js [hunk 1]
```diff
@@ -10,46 +11,77 @@ import overpassRegularWoff2 from './assets/overpass/overpass-regular.woff2'
 import overpassSemiBoldWoff from './assets/overpass/overpass-semibold.woff'
 import overpassSemiBoldWoff2 from './assets/overpass/overpass-semibold.woff2'
 
-const fontFamily = {
-  Light: [
+
```
Comment (AquiGorka) [inline src/components/BaseStyles/BaseStyles.js]: Extra line here.

File: src/components/BaseStyles/BaseStyles.js [hunk 2]
```diff
@@ -106,14 +138,4 @@ const BaseStyles = createGlobalStyle`
   }
 `
 
-BaseStyles.propTypes = {
```
Comment (AquiGorka) [inline src/components/BaseStyles/BaseStyles.js]: Why did you remove this altogether?

Comment (bpierre) [reply src/components/BaseStyles/BaseStyles.js]: I only moved it 😅 https://github.com/aragon/aragon-ui/pull/261/files/0ebfc1d17e4beebd9437eb3bef53fc786596d918#diff-8536021ce17375ae0d0eaba3e72019edR33

Comment (AquiGorka) [reply src/components/BaseStyles/BaseStyles.js]: Ahh sorry about that - 🙊

File: src/components/BaseStyles/BaseStyles.js (hunk 1 above)
Comment (bpierre) [reply src/components/BaseStyles/BaseStyles.js]: Ah it’s just the git diff not picking up the variable renaming:

image

Comment (AquiGorka) [reply src/components/BaseStyles/BaseStyles.js]: 🙊x2
